packages/augur-ui/src/modules/trading/components/form/form.styles.less

```diff
@@ -196,22 +198,22 @@
   }
 
   &.error {
-    border: 0.0625rem solid fade(@color-error-text, 50%);
+    border: 1px solid fade(@color-error-text, 50%);
 
     &:focus-within,
     &:focus {
-      border: 0.0625rem solid fade(@color-error-text, 50%);
+      border: 1px solid fade(@color-error-text, 50%);
     }
   }
 
   > span {
     background: @color-tertiary-background;
-    border-left: 0.0625rem solid @color-primary-alternate;
+    border-left: 1px solid @color-primary-alternate;
     color: @color-tertiary-text;
     font-size: @font-size-small;
     min-width: 4.28rem;
     overflow: hidden;
-    padding: 0.59375rem 0.75rem;
+    padding: 0.59375rem @size-12;
```

Comment:
replace `0.59375rem` with either `size-9` or `size-10`, currently that rem value is 9.5px so it's your call which to use but we should move away from half px in the new design